UM-USElab/gradecraft-development
# Do not show changes in "Complete" and "Student Visible" settings when showing grade history

### Status
**READY**

### Description
* Students should not be able to see changes made by an instructor to the "Student Visible" and "Complete" settings of a grade when viewing the grade history for a submission for an assignment. (i.e. viewing /assignments/\<id\> as a student, with a submission for the assignment). 
* Now changes made by an instructor in the "Complete" and "Student Visible" settings of a grade are not displayed in the grade display page for the student.

![image](https://user-images.githubusercontent.com/33792934/62375314-05059e00-b50c-11e9-86d6-4eeba9369e7f.png)
![image](https://user-images.githubusercontent.com/33792934/62375353-1c448b80-b50c-11e9-9475-feba1b95bc6f.png)

### Migrations
NO

### Steps to Test or Reproduce
1. As a student, submit an assignment. 
2. As an instructor, grade the submission for the student and make sure the grade is set to be student visible.
3. When viewing the grade as a student under the "My Submission" tab, the changes in the "Complete" and "Student Visible" settings should not be displayed in the grade history page 

### Impacted Areas in Application
* Viewing a grade for a submission as a student
* presenters/submissions/grade_history.rb

======================
Closes #3385, #3764
